static/js/markerclusterer.js

```diff
@@ -191,9 +190,7 @@ function MarkerClusterer(map, opt_markers, opt_options) {
  * @type {string}
  * @private
  */
-MarkerClusterer.prototype.MARKER_CLUSTER_IMAGE_PATH_ =
-    'http://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclusterer/' +
-    'images/m';
+MarkerClusterer.prototype.MARKER_CLUSTER_IMAGE_PATH_ = '../images/m';
```

Comment:
Just one more thing on this change:

If `markerclusterer.js` is a third-party library and if you've updated the file with the latest version of the library, we shouldn't modify it in place. If the change on this line, regarding the path to the images set in `MARKER_CLUSTER_IMAGE_PATH_` is made by you, you should revert it and leave it to the default and then set the proper value outside of this file, in another JS file.